setup.py

```diff
@@ -17,13 +17,20 @@
     author='Piotr Mitros',
     author_email='pmitros@edx.org',
     packages=['djpyfs'],
-    license = "AGPLv3",
-    url = "https://github.com/edx/django-pyfs",
-    long_description = ld,
-    classifiers = [
+    license="Apache 2.0",
+    url="https://github.com/edx/django-pyfs",
+    long_description=ld,
+    classifiers=[
         "Development Status :: 4 - Beta",
+        "Framework :: Django",
+        "Framework :: Django :: 1.8",
+        "Framework :: Django :: 1.9",
+        "Framework :: Django :: 1.10",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.5",
```

Comment:
Should also include "Programming Language :: Python :: 3", some tools judge Python 3 support based on that